src/pages/Account.vue

```diff
@@ -146,6 +146,11 @@
         },
         mounted: function () {
             Store.getVoucherPaymentState();
+            // TODO: Have a standard way of having global router messages.
```

Comment:
We should ticket this but put it low down in the list or maybe just tech debt. I would imagine that we would expand the message mixin to include some functionality which lets you set a message on the router. It only really becomes something we'll need to do if we want to send messages over the router in other places apart from here.